### DISCUSSION TITLE
Add place-anywhere config to ender crystals

### DISCUSSION THREAD

**Discussion by explorer3039** [PurpurMC/Purpur#1415] [__Completed__] [answered]
I hava an anarchy server and I want players to place crystals everywhere instead of obsidians or bedrocks, If purpur implement this, crystal pvp can be much easier.
Hope this to be implemented.

**Comment by granny** [comment, marked as answer]
implemented as of build # 2042, https://github.com/PurpurMC/Purpur/commit/ad645dcbe80e6976a09d392d29d0e906d491bdb3
